pkgs/development/web/nodejs/v6.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, openssl, python, zlib, libuv, v8, utillinux, http-parser
+, pkgconfig, runCommand, which, libtool
+, callPackage
+}@args:
+
+let
+  version = "6.0.0";
+  src = fetchurl {
+    url = "https://nodejs.org/download/rc/v6.0.0-rc.4/node-v6.0.0-rc.4.tar.gz";
```

Comment:
alright, i'll try my best - thank you for the review. Any comments on the way I split up the nix expressions in general ? Could/Should I have done that better in some other way?